CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Current Develop Branch
 
 ## 9.2.1 Thu Mar 25 2021
+- [#10692](https://github.com/MetaMask/metamask-extension/pull/10692): Prevent users from hitting the missing template error
+- [#10712](https://github.com/MetaMask/metamask-extension/pull/10712): Fix infinite spinner when attempting an approve transaction for some token contracts
```

Comment:
"some token contracts" seems a bit vague. I thought this applied to tokens with empty symbols - is there more to it than that? :thinking: 
```suggestion
- [#10712](https://github.com/MetaMask/metamask-extension/pull/10712): Fix infinite spinner when attempting an approve transaction for token with empty symbol
```

Reply:
cc @brad-decker

Reply:
I'm going to push a more precise message

Reply:
That doesn't sound right :thinking: I thought this error occurred when we did successfully retrieve the token symbol, but it was an empty string. That's what the PR description implies as well.

Reply:
This issue occurs when the `fetchSymbolAndDecimals` method returns the `DEFAULT_SYMBOL`. See https://github.com/MetaMask/metamask-extension/blob/develop/ui/app/helpers/utils/token-util.js#L81

This happens when the request for the symbol fails. It could also happen when the request for the symbol successfully returns an empty string, but I am not sure that we have ever seen that in practice.